ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/mgmt_gateway.schema.yml

```diff
@@ -0,0 +1,8 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  mgmt_gateway:
+    type: str
+    description: OOB Management interface gateway in IPv4 format.
```

Comment:
Let's add that by default it is used to configure a default route except if overriden by <mgmt_destination_networks>

Reply:
done